airbyte-cdk/python/airbyte_cdk/utils/__init__.py

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+from .traced_exception import AirbyteTracedException
```

Comment:
allows consumers to import using `from airbyte_cdk.utils import AirbyteTracedException` instead of `from airbyte_cdk.utils.traced_exception import AirbyteTracedException` and also means consumers don't break if we rename or move files

Reply:
nice 👍